Makefile

```diff
@@ -13,6 +13,9 @@ endif
 CGO_CFLAGS=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 GOBIN = build/bin
 GO ?= latest
+XGOVERSION ?= 1.9.2
+XGOIMAGE = status-im/xgo:$(XGOVERSION)
```

Comment:
We need to change this to `statusteam` as `-` is not allowed in a Docker ID. We have already `statusteam` organization created in Docker Cloud.

Reply:
@adambabik Done!

Reply:
Thanks!